aten/src/ATen/native/mkldnn/xpu/detail/Matmul.h

```diff
@@ -28,9 +28,9 @@ static inline void matmul(
       "oneDNN input matrixes must have the same ranks");
   TORCH_CHECK(result.defined(), "oneDNN matmul result should be defined");
 
-  // at::Device curDevice = at::Device(at::kXPU, current_device());
-  // auto engine = GpuEngineManager::Instance().get_engine(curDevice);
-  // auto strm = GpuStreamManager::Instance().get_stream();
+  at::Device curDevice = at::Device(at::kXPU, current_device());
```

Comment:
The code style is not aligned with other variables.
```suggestion
  at::Device cur_device = at::Device(at::kXPU, current_device());
```

Reply:
Has modified to `cur_device`, thanks.